tests/e2e/fixtures/statsFixtures.js

```diff
@@ -3,10 +3,9 @@ const dayjs = require('dayjs');
 const seasonFixtures = [
   {
     name: 'Clubs 2022',
-    // startTime: 1642642200,
-    startTime: dayjs('2022-01-19', 'yyyy-mm-dd').valueOf(),
-    // endTime: 1674178200,
-    endTime: dayjs('2022-02-18', 'yyyy-mm-dd').valueOf(),
+    startTime: dayjs('2022-01-19 20:30').valueOf(),
+    // 4 weeks long, minus 2.5 hours
+    endTime: dayjs('2022-02-16 18:00').valueOf(),
```

Comment:
Updating the season start and end times to be just under 4 weeks long is how I got the test case to replicate the failure